### DISCUSSION TITLE
Only 8x Full nodes connected since 1.1.6

### DISCUSSION THREAD

**Discussion by tydeno** [Chia-Network/chia-blockchain#6079] [Support]
Hei there

I learned that It is important for proper farming to have atleast 20x full-nodes connected. However, I got like 6-8 max at all times. Prior to 1.1.6 I had 30-50 connections going round. Since 1.1.6 its way less.

I tried so far: 
- Adding the available introducers and nodes

Is there anything else I could possibly do to get more connections fired up?

Thx for your support.

**Comment by tiny0little** [comment]
inbound ```tcp8444``` opened?

**Reply by tydeno** [reply]
Yes. Config has not changed since 1.1.5 from my side.

**Reply by tydeno** [reply]
Its a single full node on my LAN with upnp set to true in cfg. Port forwarding on my Router is set correctly. Telnetting from external to my WAN IP on Port 8444 works flawless.

**Reply by tiny0little** [reply]
as you mentioned, I simply add the introducers and well known nodes on regular basis via cron

something like this

```
chia show -a node.chia.net:8444 > /dev/null 2> /dev/null && sleep 5 && \
  chia show -a introducer-apne.chia.net:8444 > /dev/null 2> /dev/null && sleep 5 && \
    chia show -a introducer-apse.chia.net:8444 > /dev/null 2> /dev/null && sleep 5 && \
      chia show -a introducer-or.chia.net:8444 > /dev/null 2> /dev/null && sleep 5 && \
        chia show -a introducer-va.chia.net:8444 > /dev/null 2> /dev/null && sleep 5 && \
          chia show -a introducer-eu.chia.net:8444 > /dev/null 2> /dev/null && sleep 5 && \
            chia show -a node.chia.net:8444 > /dev/null 2> /dev/null &
```
I noticed that introducer and/or node often not added, possibly due to large number of connections on the other end, also once introducers are gone from the list, number of connections slowly decreases

**Comment by joelrb** [comment]
> Hei there
> 
> I learned that It is important for proper farming to have atleast 20x full-nodes connected. However, I got like 6-8 max at all times. Prior to 1.1.6 I had 30-50 connections going round. Since 1.1.6 its way less.
> 
> I tried so far:
> 
> * Adding the available introducers and nodes
> 
> Is there anything else I could possibly do to get more connections fired up?
> 
> Thx for your support.

What's the status of your nodes? I got the same issue. I only get eight nodes everyday. Adding peers and introducers doesn't' work. Tried in another computer still the same. Chia ver 1.1.6. Port 8444 open in router and forwarding to correct local machine.

**Reply by tydeno** [reply]
Thanks for all your replies. It turned out, that the automatically created rule using Upnp IGD got lost somehow on my Router. 
Thats why I ended up only having 6x nodes. As soon as I set the rule again, nearly every second a node poped up.

**Comment by happycouak** [comment]
Do you mean by "atleast 20x full-nodes connected" that for the same pool of plots you have 20x full node running on the same network / behind the same public IP ?

**Reply by tydeno** [reply]
No just one. I'm talking about other full-nodes on the chia network connected to my full-node.

**Comment by BrandtH22** [comment]
@tydeno , based on your previous message it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
